packages/app-admin-ui/index.js

```diff
@@ -176,33 +216,36 @@ class AdminUIApp {
     const _app = express.Router();
 
     if (this.authStrategy) {
-      _app.use(this.createSessionMiddleware(keystone));
+      _app.use(this.createSessionMiddleware());
     }
 
     _app.use(this.adminPath, app);
 
     if (this.enableDefaultRoute) {
       // Attach this last onto the root so the `this.adminPath` can overwrite it
       // if necessary
-      _app.get('/', (req, res) => res.sendFile(path.resolve(__dirname, './server/default.html')));
+      _app.get('/', (req, res) => {
+        if (!this.isAccessAllowed(req)) {
+          res.sendFile(path.resolve(__dirname, './server/default.html'));
```

Comment:
We should still send this even when the user is allowed access; if they deployed with the defaults of `/admin`, but have `/` bookmarked, they'll then be shown a blank screen, for example.